### DISCUSSION TITLE
Steam Deck Port

### DISCUSSION THREAD

**Discussion by ozwaldorf** [catppuccin/catppuccin#1928] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

Steam Deck UI is built with React and can be themed using the DeckyLoader with the CSSLoader plugin installed (https://deckthemes.com/)

### Has the requested port been themed already?

Mostly complete, there are some edge case colors and some spots that can be refined. Feel free to test out the theme so far:
https://github.com/ozwaldorf/catppuccin-steam-deck

### Any additional comments?

> Open to PRs from anyone willing to help!

- [ ] TODO: readme previews
- [ ] TODO: fix menu selected item
- [ ] TODO: submit to https://deckthemes.com

**Comment by sgoudham** [comment]
Closing as https://github.com/catppuccin/steam-deck exists now! 🎉
